### ISSUE TITLE
Game Image Opacity Does Not Work Properly

### ISSUE THREAD

**Issue by ThereAreMorePenisMan** [Jonius7/SteamUI-OldGlory#11]
**Thanks for the updates** 

I think this screenshot can explain everything
![dim](https://user-images.githubusercontent.com/78355978/107146130-745b1a00-695b-11eb-9f80-131ccc22b182.PNG)
every game with a icon in front of it will be considered as an uninstalled game
can you also add another experimental option for removing game previews

**Comment by Jonius7** [comment]
For removing game previews: this is on the way with a small auto-update coming soon (next 24 hrs). 

Next time you open old_glory up. You'll be able to change the `--RemoveGameHover` value to `none` in the `variables.css` file. The GUI is already full so, it won't be on the CSS Options Page just yet.

As for the game image dimming, I am not finding a quick solution to this that is CSS only. It may require JavaScript. You may be better off selecting the `Default (Full Opacity)` value for now.

**Comment by Jonius7** [comment]
I was able to get it to dim the correct images behind an icon/text tag, however, it comes with the ugly "broken image" pop-ups in the top left corner for a second or two. I would not find that acceptable, so the change is disabled by default.

In `_home_collections.scss`

```css
/* More precise uninstalled game image opacity but "broken images" annoyance - disabled by default */
.appportrait_Draggable_1pwP4 > :nth-last-of-type(3) {
  //opacity: var(--UninstalledGameImageOpacity);
}
```
the // is commenting it out at the moment

**Comment by ThereAreMorePenisMan** [comment]
I tested the latest small updates and I'm not sure what do you mean by ugly pop ups because there was no "broken image" pop-ups at all.

Here is a short video of how the new dimming looks on my computer
(I'm using **Threshold+shiina** skins i did test this with the normal portrait mode still no broken image):

https://user-images.githubusercontent.com/78355978/107883602-1d2dea00-6f05-11eb-93b5-821b101eca36.mp4

Although there are complicated and hard to explain issues,
Like this one:
![Screenshot (8)_LI](https://user-images.githubusercontent.com/78355978/107885434-8dda0400-6f0f-11eb-86b0-1d76238531ce.jpg)

No pressure :) also thanks for the game preview removal update.

**Comment by Jonius7** [comment]
I would try adding in:

```css
.appportrait_FriendsBar_3dncO {
    opacity: 1;
}
```
below the other `UninstalledGameImageOpacity` code

Unfortunately I cannot test this myself, as I don't have a friend playing a game that I own but I also have it uninstalled.

**Comment by Jonius7** [comment]
![2021-07-03 00_40_40-DevTools - Steam](https://user-images.githubusercontent.com/1049415/124296836-bfd9c300-db9d-11eb-9f39-1d8942689e69.png)
Unable to completely solve this with CSS only in the case of Friends icons appearing. Since they take up the same `div` slot as the `uninstall icon` would be, it uses the UninstalledGameOpacity. And there is no way for it to select the correct div while also checking to see if its sibling is a `.appportrait_FriendsBar_3dncO` as well.

A JS fix could be possible to switch the ordering of divs around, may be ugly though.

**Comment by Jonius7** [comment]
A new JS Tweak should be able to fix this behaviour now pretty consistently df693bc

**Comment by Jonius7** [comment]
This is now fixed and fully implemented as a JS Tweak with Release 5.7.3.
https://github.com/Jonius7/SteamUI-OldGlory/releases/tag/Release_5.7.3

If you want to use the CSS only tweak that was previously used, it is commented out in `scss\_home_collections.scss` under the 
`/* Grid - Uninstalled game image opacity */` section.

I will leave this issue open for a little while.
